executor/util/envUtils.go

```diff
@@ -49,7 +49,7 @@ func GetGlobalEnvVariables(cicdRequest *helper.CiCdTriggerEvent) (map[string]str
 		envs["APP_NAME"] = cicdRequest.CommonWorkflowRequest.AppName
 		envs["TRIGGER_BY_AUTHOR"] = cicdRequest.CommonWorkflowRequest.TriggerByAuthor
 		envs["DOCKER_IMAGE"] = image
-
+		envs["JOB_NAME"] = cicdRequest.CommonWorkflowRequest.AppName
```

Comment:
expose env variables conditionally based on job app name type and devtron app name, use IsJob in workflow event struct